Apps/WebClient/src/ClientApp/src/components/calendar/body.vue

```diff
@@ -333,7 +333,7 @@ export default class CalendarBodyComponent extends Vue {
         if (type == EntryType.Medication) {
             return (entry as MedicationTimelineEntry).medication.brandName;
         } else if (type == EntryType.Immunization) {
-            return (entry as ImmunizationTimelineEntry).immunization.name;
+            return (entry as ImmunizationTimelineEntry).immunization.location;
```

Comment:
Are we displaying the location on the calendar title? that looks weird, could the type maybe be better?

Reply:
good question, mbe @ninosamson can confirm

Reply:
This looks odd, why was it changed?